lib/choice.rb

```diff
@@ -0,0 +1,26 @@
+class Choice
+
+  def first_choice(first_choice)
```

Comment:
I'd avoid using the same name for a method and a parameter; I try to avoid using the same name anywhere actually.

Reply:
I think we might not use choice anywhere, is that correct? If not, this class and its tests could be removed?